components/Card/Card.css

```diff
@@ -0,0 +1,87 @@
+/* 
+    Author Name:Nikhil Nirmal,
+    color pallet used:
+    background:#0a192f;
+    font 1 : #55dabe;
+    font 2 : #ccd6f6;
+    font 3 : #6e7996;   
+*/
+:root {
+    --card-white : white;
+}
+
+.card{
+    height: max-content;
+    width:18.75rem;
+    border-radius: 2px;
+}
+
+.card-ecommerce{
+    width:18.75rem;
+    background-color: var(--card-white) ;
+}
+
+.card-title{
+    font-size: 1.5rem;
+    font-weight: 500;
+    padding: .5rem 1rem 0;
+    background-color: var(--card-white) ;
+}
+
+.card-text{
+    font-size: 1rem;
+    padding: 1rem;
+    background-color: var(--card-white) ;
+}
+
+.card-image{
+    background-color: var(--card-white) ;
+    border-radius: 2px 2px 2px 2px;
+    background-size: contain;
+    width: 18.75rem;
+    height: auto;
+}
+
+.card-head{
+    background-color: var(--card-white) ;
+    padding: 10px 0;
+    display: flex;
+    justify-content: space-between;  
+    align-items: center; 
+    padding: 1rem;
+}
+
+.product-image{
+    width: 18.75rem;
+    height: auto;
+}
+
+.prices {
+    background-color: var(--card-white) ;
+    padding: 0px 1rem 1rem 1rem;
+}
+
+.current-price{
+    font-weight: 400;
+    padding-right: 5px;
+    background-color: var(--card-white) ;
+}
+
+.before-price{
+    font-weight: 300;
+    background-color: rgba(182, 182, 182, 0.432);
+    padding-right: 5px;
+    text-decoration: line-through;
+}
+
+.discount{
+    margin-left:0.5rem ;
+    font-weight: 400;
+    font-size: 1rem;
+    color: green;
+    background-color: var(--card-white) ;
+}
+
+button {
```

Comment:
Try grabbing elements via class name and not element name

Reply:
Yes will take note of that